system-addon/content-src/components/Card/_Card.scss

```diff
@@ -151,19 +151,21 @@
     display: flex;
     font-size: 11px;
     left: 0;
-    padding: 12px 16px 12px 14px;
+    padding: 9px 16px 9px 14px;
     position: absolute;
     right: 0;
   }
 
   .card-context-icon {
+    display: inline-block;
```

Comment:
Shouldn't this property come from `.icon`?

Reply:
oh very true. it was already set to that. thanks!